PWGMM/Mult/Tasks/CMakeLists.txt

```diff
@@ -19,6 +19,11 @@ o2physics_add_dpl_workflow(dndeta
                     PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
                     COMPONENT_NAME Analysis)
 
+o2physics_add_dpl_workflow(dndetajbae
```

Comment:
Please use something like `dndeta-hi` instead.

Reply:
I just changed file name and CMakeLists. Thank you!

Reply:
Very good, thank you. You still have to fix the formatting, however, please look at what the CI tests complain about. The places with formatting issues are marked directly in diff that you can see in `Files changed` tab.